src/index.html

```diff
@@ -11,7 +11,155 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <label class="form_field">
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="form_field">
+          Name:
+          <input
+            type="text"
+            name="name"
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="form_field">
+          How old are You?
+          <input
+              type="number"
+              value="12"
+              name="age"
+              min="1"
+              max="100"
+          >
```

Comment:
pay attention to attribute formatting
```suggestion
          <input
            type="number"
            value="12"
            name="age"
            min="1"
            max="100"
          >
```